classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] It would be nice if you make the students just readable and not accessible because it should be updated implicitly and by the app only but not externally :thinking:

Kindly make your getters and setters like below :arrow_down:

```ruby
attr_accessor :label
attr_reader :students
```

Reply:
done